gcovr/gcov_parser.py

```diff
@@ -480,25 +479,24 @@ def _report_lines_with_errors(
     lines = [line for line, _ in lines_with_errors]
     errors = [error for _, error in lines_with_errors]
 
-    context.logger.warn(
-        "Unrecognized GCOV output for {source}\n"
-        "\t  {lines}\n"
-        "\tThis is indicative of a gcov output parse error.\n"
-        "\tPlease report this to the gcovr developers\n"
-        "\tat <https://github.com/gcovr/gcovr/issues>.",
-        source=context.filename,
-        lines="\n\t  ".join(lines),
+    lines_output = "\n\t  ".join(lines)
+    logger.warning(
+        f"Unrecognized GCOV output for {context.filename}\n"
+        f"\t  {lines_output}\n"
+        f"\tThis is indicative of a gcov output parse error.\n"
+        f"\tPlease report this to the gcovr developers\n"
+        f"\tat <https://github.com/gcovr/gcovr/issues>."
```

Comment:
```suggestion
    logger.warning(
        f"Unrecognized GCOV output for {context.filename}\n"
        "\t  {'\n\t  '.join(lines)\n"
        "\tThis is indicative of a gcov output parse error.\n"
        "\tPlease report this to the gcovr developers\n"
        "\tat <https://github.com/gcovr/gcovr/issues>."
```

Reply:
That change will not work. Using escape sequences inside a f-string at this place is not allowed according to Python spec. I tried that already.

Reply:
Ok. Please remove the f from the strings which don't contain a reference.

Reply:
In my experience, I either assume that all string pieces are f-strings or none. In other words, this proofs against the bug, where someone adds some format string in the normal string pieces and forgets to change them into f-strings.

If you still insist, I will change it.

Reply:
There is a flake8 check which forbids f-strings where they aren't needed.

Reply:
Done.